### ISSUE TITLE
Consider strong name signing Nuget packages

### ISSUE THREAD

**Issue by steveellis** [lostromb/concentus#20]
Right now the Concentus.dll and Concentus.Oggfile.dll need to be signed together if you're using them in a deployment that is signed. Otherwise you get an error like:

> Could not load file or assembly 'Concentus, Version=1.1.6.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044) 

It's compounded by the fact that if you're using Concentus.dll and Concecntus.Oggfile.dll together, you have to sign them together to make the dependency work.

Happy to update the ticket with more info about how I signed them in my deployment or provide links to strong name signing assemblies if that will help.

**Comment by steveellis** [comment]
For other people encountering this, I've been able to sign the files and overcome the error mentioned above by signing the assemblies with this project:

https://github.com/brutaldev/StrongNameSigner

I have to include the signed versions in my project directly without nuget.

I think it may be limited to using Concentus in a Windows EV signed project, which is how I'm deploying to windows clients. I've now deployed Concentus in Android, iOS and macOS, each of which is signed in their own special ways of course, without any problems.
